resources/assets/less/app/kanban.less

```diff
@@ -87,20 +197,21 @@
       padding: 1px 0 0;
       position: relative;
       vertical-align: top;
-      background: #f5f5f5;
+      background: transparent;
 
       .heading {
-        background: #fff;
-        font-size: 19px;
-        border-bottom: 5px solid #f5f5f5;
+        background: rgba(65,63,79,0.14);
+        font-size: 18px;
+        //border-bottom: 5px solid #f5f5f5;
```

Comment:
that is ok :) my comments are not to criticise. I like to keep code lean as much as possible.

There is lots of thing in this project that I would like to improve. I like your suggestions, and would like to implement them.